templates/default/stunnel.conf.erb

```diff
@@ -99,4 +99,7 @@ TIMEOUTclose = <%= opts['timeout_close'] ? 1 : 0 %>
 <% unless opts['client'].nil? -%>
 client = <%= opts['client'] ? "yes" : "no" %>
 <% end -%>
+<% unless opts['verify_chain'].nil? %>
```

Comment:
Little nitpick, but I'd like you to add the whitespace chomp at the end of the tag like the other unless blocks for consistency. At the end change `%>` to `-%>` on this line and the end line as well.